### DISCUSSION TITLE
How to see electrode stoichiometries?

### DISCUSSION THREAD

**Discussion by Jadhav-Sourabh** [pybamm-team/PyBaMM#3515] [General]
### PyBaMM Version

Latest

### Python Version

3.11

### Describe the bug

How can I see the values of stoichiometries of LFP and Gr?
Modify?

### Steps to Reproduce

NA

**Comment by rtimms** [comment]
Could you elaborate? Do you want to see min/max stoichiometries or look at the stoichiometry as a function of time from the results? 

Converting this to a discussion.

**Comment by Jadhav-Sourabh** [comment]
I want to see this value from parameter values itself not from result. 

Wants to use this value for another calculation.

**Reply by rtimms** [reply]
```python3
import pybamm

parameter_values = pybamm.ParameterValues("Chen2020")
x0, x100, y100, y0 = pybamm.lithium_ion.get_min_max_stoichiometries(parameter_values)
```

**Reply by Jadhav-Sourabh** [reply]
Are you using chen2020 stoichiometries for prada2013 ?

**Reply by rtimms** [reply]
No, that was just an example. You can run the above with any parameter set you like

```python3
import pybamm

parameter_values = pybamm.ParameterValues("Prada2013")
x0, x100, y100, y0 = pybamm.lithium_ion.get_min_max_stoichiometries(parameter_values)
print(x0, x100, y100, y0)
```

**Reply by Jadhav-Sourabh** [reply]
Getting error while using for 'Prada2013'

**Reply by brosaplanella** [reply]
What error do you get? Can you share the full error message here?

**Reply by Jadhav-Sourabh** [reply]
`pybamm.expression_tree.exceptions.SolverError: could not find acceptable solution`

**Reply by brosaplanella** [reply]
That's strange, it is running fine at my end. Can you confirm which PyBaMM version are you using?

**Reply by Jadhav-Sourabh** [reply]
23.5

**Reply by Jadhav-Sourabh** [reply]
If possible can you please share the stoichiometries values ?

**Reply by brosaplanella** [reply]
`0.017619411644243348 0.8100363695199028 0.003767884398705004 0.703500714165226`

**Reply by Jadhav-Sourabh** [reply]
Thanks

**Reply by Jadhav-Sourabh** [reply]
Which PyBaMM version?

**Reply by brosaplanella** [reply]
Latest: 23.9
